ios/AllAboutOlaf.xcodeproj/project.pbxproj

```diff
@@ -1422,7 +1422,7 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				HEADER_SEARCH_PATHS = "$(inherited)";
-				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.2;
```

Comment:
Are we good to bump min iOS to 16.2 from 15.0? xcodebuild only lists 16.2 devices with Xcode 14.2.

Reply:
Sounds good to me